app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -63,12 +63,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     }
 
     private void loadMediaInfo() {
-        if(imgvCover == null) {
```

Comment:
I'm thinking we probably shouldn't rely on onCreateView() being called before onStart().